config/config.go

```diff
@@ -70,17 +69,21 @@ func LoadClient(ctx context.Context, env string) (Client, error) {
 		return Client{}, fmt.Errorf("processing config: %w", err)
 	}
 
+	md := d.Environments[env].Metadata
 	if cfg.Google.ApplicationCredentials == "" {
-		if creds, ok := ctx.Value(CredsKey).(string); ok {
+		if creds := md.ApplicationCredentials; creds != "" {
 			cfg.Google.ApplicationCredentials = creds
 		}
 	}
-	md := d.Environments[env].Metadata
-	bucket := md.Bucket
-	if bucket != "" {
+
+	if bucket := md.Bucket; bucket != "" {
 		cfg.Google.StorageBucket = bucket
-		cfg.Google.StoragePrefix = md.Location
 	}
+
+	if location := md.Location; location != "" {
```

Comment:
This is sortof optional, It's better to refactor or change code not relevant to the issue/story in a separate PR.